config/webpack.prod.js

```diff
@@ -171,7 +171,7 @@ module.exports = function (env) {
      * See: http://webpack.github.io/docs/configuration.html#plugins
      */
     plugins: [
-      new MiniCssExtractPlugin({ filename: '[name]-[hash].css', chunkFilename: '[name]-[chunkhash].css' }),
+      new MiniCssExtractPlugin({ filename: '[name].css', chunkFilename: '[name]-[chunkhash].css' }),
       new HashedModuleIdsPlugin(),
       new webpack.LoaderOptionsPlugin({
```

Comment:
does `postcss` option depend on the MiniCssExtractPlugin?  the one in webpack.dev.js does not contain this option.

Reply:
MiniCssExtractPlugin is only used in prod configuration

Reply:
sorry if i'm not clear.  my question is whether there is a dependency between options in the  LoaderOptionsPlugin and the miniExtractPlugin.  To be more specific if you changed a config in MiniCssExtractPlugin does that break the autoprefixer thing that's in the LoderOptionsPlugin?